examples/platforms/da15000/tool.h

```diff
@@ -0,0 +1,140 @@
+/**
+ * TOOL abstraction -- common coding convention for multiple toolchains:
```

Comment:
Is this file necessary?  We have a minimal version in `include/platforms/tool.h`.  The cc2538 example does not use this.